source/elements/atoms/Grid/README.md

```diff
@@ -0,0 +1,24 @@
+# Grid
+
```

Comment:
README could use explicit language as to what Grid is. (makes Grid consistent with all other Baseline components).